contracts.json

```diff
@@ -90,22 +90,22 @@
   },
   "mumbai": {
     "BridgedPolygonNORI": {
-      "proxyAddress": "0xB3fe45C08137dD6adACb2918D899e0C0dBB036C8"
+      "proxyAddress": "0xd768Df6475a853194dB4afF2dDbDC0ea29F05c66"
```

Comment:
these may have changed due to Scott's original activity on this branch, in which I'm assuming he correctly redeployed proxies + implementations...
I only ever used one deployment command, which was `hardhat deploy --network mumbai --tags market`

This is actually making me curious about how my deployments would have broken CI, because I would have thought they would have upgraded the new proxy addresses that were created here as opposed to the existing ones that Defender currently relies on.  I'm not sure.

I suppose we also need to decide if we want to merge these proxy addresses or not.

Reply:
Correct, I force deployed new proxies since I was trying to start with fresh state.  Wasn't necessary for BPNori but that flag is all or nothing and the dependency graph in the deployment scripts pulled this in.